### DISCUSSION TITLE
Workflow - How to manage bigger notes

### DISCUSSION THREAD

**Discussion by jayrajput** [ahyatt/ekg#32] [Q&A]
Looking for comments on how one shall manage bigger notes. For smaller notes, `ekg` works great. But for things which are big, shall one put it in a new file and just add the links to the `ekg`. Looking for guidance on this.

**Comment by ahyatt** [comment]
You might want to check out https://github.com/ahyatt/ekg/pull/19, which solves this problem.  However, I'd say it's better to have multiple small notes than a few big notes.  Smaller notes give you something more specific to link to, and the tags will be more specific.

If you look at the `inlines` branch, we have a way of embedding files, which provides another solution - although I'd say it isn't a good solution if you just want a big note, and don't really care about it being a file.

**Comment by jayrajput** [comment]
Rather than big note, I would like to articulate it as final document/note which I am working on.

So `ekg` is great with small notes, but there is a friction for working with the final document. One friction is that you cannot quickly jump to it using a title. I wonder if the concept of `document/` kind of covers that.

Let me explain with an example. I create these design documents for my team. My design documents may be a collection of multiple small documents which are already present in my `ekg` notes tagged nicely and searchable. I will start this design document by combining the small documents already present in my `ekg` notes. Now I have this document started, I would like to keep going back to this document, sharing with my team, reviewing it together. 

I do want to keep this document in my reference system, but I would like to jump directly to it rather than searching the tag and then opening it. The `document/` concept with browse may work.

One thought I can use another system all together like `denote` to manage such document and jump quickly to this document using `denote`.

Looking for inputs how others manage such workflow.
